config/config.exs

```diff
@@ -21,8 +21,10 @@ config :logger, :console,
 
 
 config :kafka_ex,
-  brokers: [{"192.168.99.100", 9092}],
-  consumer_group: "kafka_ex",
+  brokers: System.get_env("KAFKA_BROKERS") # expected to be comma separated list of broker_host:port
+            |> String.split(",")
+            |> Enum.map(fn(broker)-> List.to_tuple(String.split(broker, ":")) end ),
+  consumer_group: System.get_env("KAFKA_CONSUMER_GROUP"),
```

Comment:
@ashkan18 Oooooo `KAFKA_CONSUMER_GROUP` is also good - we should set different group names per environment...